tools/configen/configen/configen.py

```diff
@@ -128,15 +128,44 @@ def is_incompatible(type_: Type[Any]) -> bool:
     return True
 
 
+def get_default_flags(module: ModuleConf) -> List[Parameter]:
+
+    def_flags: List[Parameter] = []
+
+    if module.default_flags._convert_ is not None:
+        def_flags.append(
+            Parameter(
+                name="_convert_",
+                type_str="str",
+                default=f"""\"{str(module.default_flags._convert_.name)}\"""",
```

Comment:
1. name is already a string.
2. no need for a multiline string:

```suggestion
                default=f"\"{module.default_flags._convert_.name}\",
```

Reply:
good catch. This was left over from the previous formatting requirements.